content/docs/secure/encryption/deployment.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "Deployment"
+linkTitle: "Deployment"
+weight: 1
+Description: >
+  Deployment.
+---
+Encryption is enabled as part of the Dell CSI driver installation. The drivers can be installed either by a Helm chart or by the Dell CSI Operator. 
+In the tech preview release, Encryption can only be enabled via Helm chart installation.
+
+Except for additional Encryption related configuration outlined on this page, 
+the rest of the deployment process is described in the correspondent [CSI driver documentation](../../../csidriver/installation/helm).
+
+## Vault Server
+
+Hashicorp Vault must be [pre-configured](../vault) to support Encryption. The Vault server's IP address and port must be accessible 
+from the Kubernetes cluster where the CSI driver is to be deployed.
+
+## Helm Chart Values
+
+The drivers that support Encryption via Helm chart have an `encryption` block in their *values.yaml* file that looks like the following:
+
+```yaml
+encryption:
+  # enable: Enable/disable volume encryption feature
+  # Encryption cannot be enabled in tandem with resizer or authorization.
+  # Enable this feature only after contact support for additional information.
+  enabled: false
+
+  # pluginName: The name of the provisioner to use when encryption is enabled.
+  pluginName: "sec-isilon.dellemc.com"
+
+  # image: Encryption driver image name.
+  image: "dellemc/csm-encryption:v0.1.0"
+
+  # imagePullPolicy: If specified, overrides the chart global imagePullPolicy.
+  imagePullPolicy:
+
+  # logLevel: Log level of the encryption driver.
+  # Allowed values: "error", "warning", "info", "debug", "trace"
+  logLevel: "error"
+  
+  # livenessPort: HTTP liveness probe port number. 
+  # Leave empty to disable the liveness probe.
+  # Example: 8080
+  livenessPort:
+
+  # extraArgs: Extra command line parameters to pass to the encryption driver.
+  # Allowed values:
+  # --sharedStorage - may be required by some applications to work properly.
+  # When set, performance is reduced, and hard links cannot be created.
+  # See the gocryptfs documentation for more details.
+  extraArgs: []
+```
+
+| Parameter | Description | Required | Default |
+| --------- | ----------- | -------- | ------- |  
+| enabled | Enable/disable volume encryption feature. | No | false |
+| pluginName | The name of the provisioner to use when encryption is enabled. | No | "sec-isilon.dellemc.com" |
+| image | Encryption driver image name. | No | "dellemc/csm-encryption:v0.1.0" |

```

Comment:
this should be v1.4.0

Reply:
After a discussion with @sharmilarama it was decided to keep v0.1.0.